news/2020-10-29-project-loved-october-2020.md

```diff
@@ -0,0 +1,232 @@
+
+## <a id="taiko"></a>osu!taiko
+
+osu!taiko Loved candidates were chosen by [-Kazu-](https://osu.ppy.sh/users/920861), [D3kuu](https://osu.ppy.sh/users/7807444), [nyanmi-1828](https://osu.ppy.sh/users/6866480) and [Raphalge](https://osu.ppy.sh/users/3918650)!
+
+---
+
+[![](/wiki/shared/news/2020-10-29-project-loved-october-2020/taiko/1-strong-280.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[katagiri - STRONG 280](https://osu.ppy.sh/beatmapsets/1201900#taiko) by [KTYN](https://osu.ppy.sh/users/2250574) and [X a v y](https://osu.ppy.sh/users/3738344)\
+280 BPM, 4:14 | 8.06★\
+*description written by [-Kazu-](https://osu.ppy.sh/users/920861)*
+
+This month we have katagiri's *STRONG 280*, a highly requested map by KTYN and X a v y, who managed to blend their styles together perfectly for an incredibly comfy 280 BPM map. KTYN is already known for these types of maps, so this stands out more for its stamina-heavy patterning that drains you throughout the whole song. It also helps a lot how catchy the song is and is a nice surprise to see a breakcore-ish song have a stamina-based style rather than a tech focus, so while this is definitely a difficult map for most players, it's also one of the most fun maps we've had a chance to feature in Loved.
+
+[![](/wiki/shared/news/2020-10-29-project-loved-october-2020/taiko/2-vtuber-mads.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Various Artists - VTuber MADs](https://osu.ppy.sh/beatmapsets/1138635#taiko) by [X a v y](https://osu.ppy.sh/users/3738344)\
+98 – 280 BPM, 4:09 | 3.19★ – 8.19★\
+The \[spinner\] difficulty is *not* being nominated for Loved.\
+*description written by [Raphalge](https://osu.ppy.sh/users/3918650)*
+
+Everyone's seen them... you, your friends, your mom, probably your dog I don't know your life. Well, no matter your opinion one thing is clear. VTubers are here to stay. They're all over your Twitter feed, in your Discord chats, and maybe there's even one behind you right now! With outdoor activities being discouraged, I suppose the virtual world takes over. So... why not spread the VTuber craze over to osu! too? Funny you should ask, because it seems our friend X a v y has gone above and beyond in doing just that! With a compilation mapset of *50* different VTuber MAD songs, they've got 99% of the playerbase set up with difficulties covering 3\* and everything up to 8\*. Looking at your already overwhelming support, you've clearly shown us that you love your VTubers. How about showing it again?
```

Comment:
```suggestion
Everyone's seen them... you, your friends, your mom, probably your dog... I don't know your life. Well, no matter your opinion one thing is clear: VTubers are here to stay. They're all over your Twitter feed, in your Discord chats, and maybe there's even one behind you right now! With outdoor activities being discouraged, I suppose the virtual world takes over. So... why not spread the VTuber craze over to osu! too?

Funny you should ask, because it seems our friend X a v y has gone above and beyond in doing just that! With a compilation mapset of *50* different VTuber MAD songs, they've got 99% of the playerbase set up with difficulties covering 3\* and everything up to 8\*. Looking at your already overwhelming support, you've clearly shown us that you love your VTubers. How about showing it again?
```